doc/classes/SpinBox.xml

```diff
@@ -71,8 +71,98 @@
+			Up button icon modulation color.
+		</theme_item>
+		<theme_item name="up_pressed_icon_modulate" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Up button icon modulation color, when the button is being pressed.
+		</theme_item>
+		<theme_item name="buttons_vertical_separation" data_type="constant" type="int" default="0">
+			Vertical separation between the up and down buttons.
+		</theme_item>
+		<theme_item name="buttons_width" data_type="constant" type="int" default="16">
+			Width of the up and down buttons. If smaller than any icon set on the buttons, the said icon may overlap neighboring elements, unless [theme_item set_min_buttons_width_from_icons] is enabled.
+		</theme_item>
+		<theme_item name="field_and_buttons_separation" data_type="constant" type="int" default="2">
+			Width of the horizontal separation between the text input field ([LineEdit]) and the buttons.
+		</theme_item>
+		<theme_item name="set_min_buttons_width_from_icons" data_type="constant" type="int" default="1">
+			If not [code]0[/code], the minimum button width corresponds to the widest of all icons set on those buttons, even if [theme_item buttons_width] is smaller.
+		</theme_item>
+		<theme_item name="down" data_type="icon" type="Texture2D">
+			Down button icon, displayed in the middle of the down (value-decreasing) button.
+		</theme_item>
+		<theme_item name="down_disabled" data_type="icon" type="Texture2D">
+			Down button icon when the button is disabled.
+		</theme_item>
+		<theme_item name="down_hover" data_type="icon" type="Texture2D">
+			Down button icon when the button is hovered.
+		</theme_item>
+		<theme_item name="down_pressed" data_type="icon" type="Texture2D">
+			Down button icon when the button is being pressed.
+		</theme_item>
+		<theme_item name="up" data_type="icon" type="Texture2D">
+			Up button icon, displayed in the middle of the up (value-increasing) button.
+		</theme_item>
+		<theme_item name="up_disabled" data_type="icon" type="Texture2D">
+			Up button icon when the button is disabled.
+		</theme_item>
+		<theme_item name="up_hover" data_type="icon" type="Texture2D">
+			Up button icon when the button is hovered.
+		</theme_item>
+		<theme_item name="up_pressed" data_type="icon" type="Texture2D">
+			Up button icon when the button is being pressed.
+		</theme_item>
 		<theme_item name="updown" data_type="icon" type="Texture2D">
-			Sets a custom [Texture2D] for up and down arrows of the [SpinBox].
+			Single up/down graphic for both up/down buttons. It is displayed in the middle of the buttons and does not change upon interaction. It is recommended to use individual [theme_item up] and [theme_item down] graphics for better usability. This can also be used as additional decoration between the two buttons.
+		</theme_item>
+		<theme_item name="down_background" data_type="style" type="StyleBox">
+			Background graphic of the down button.
```

Comment:
All of these "graphic" below could be replaced by "style" (I've never seen these being referred as "graphic") 
```suggestion
			Background style of the down button.
```

Reply:
See my comment on July 21 above for elaboration on how it works. Seems I cannot easily make a link to it.

Reply:
Regarding set_min_buttons_width_from_icons, see my comment above on Jul 21 for explanation. Seems tricky to make a direct link to it.